yaml/pois.yaml

```diff
@@ -1576,11 +1593,27 @@ filters:
     output:
       <<: *output_properties
       kind: {col: highway}
+  # memorial plaques more specific than plain memorials
+  - filter:
+      any:
+        - {historic: memorial, memorial: plaque}
+        - historic: memorial_plaque
+    min_zoom: 16
+    output:
+      <<: *output_properties
+      kind: plaque
+  # plain memorial
   - filter: {historic: memorial}
     min_zoom: 17
     output:
       <<: *output_properties
       kind: memorial
+  # obelisk - but not a monument or a memorial
```

Comment:
I think obelisk should be sorted above plaque and memorial – it's the thing visible from far while the others are generally attached to an obelisk?

Reply:
OK. Done in bfb655a. Moved the `monument`/`memorial` info to `kind_detail` and updated documentation and added tests.